usr/share/rear/restore/VEEAM/default/500_restore.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Agent registration and restore command
+LogPrint "Deleting Veeam SQLite Linux Agent Database"
+rm -rf /var/lib/veeam/*
+
+LogPrint "Starting veeamservice agent for linux"
+systemctl start veeamservice
+StopIfError "Failed to start veeamservice Agent for Linux"
+sleep 5
```

Comment:
What is the purpose to sleep hardcoded 5 seconds
after 'systemctl start veeamservice' succeeded?

I guess after 'systemctl start veeamservice' returns
it takes some time until veeamservice is actually ready to operate.
If this is the reason to sleep 5 seconds then I wonder
if sleeping hardcoded 5 seconds sufficiently fail-safe
to ensure that veeamservice is actually ready to operate.
Isn't there a better test to actually test
that veeamservice is actually ready to operate?